### PR TITLE
tomer shvadron/feat/backoffice ui changes

### LINKED ISSUE #1332: backoffice ui changes

### PR DESCRIPTION
**Pull request by tomer-shvadron**
### Description
A lot of small UI changes in the backoffice

### CODE REVIEW

File: apps/backoffice-v2/src/common/components/atoms/AssignDropdown/AssignDropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+import { CheckSvg, DoubleCaretSvg, UnassignedAvatarSvg } from '../icons';
+import { DropdownMenu } from '../../molecules/DropdownMenu/DropdownMenu';
+import { DropdownMenuItem } from '../../molecules/DropdownMenu/DropdownMenu.Item';
+import { DropdownMenuTrigger } from '../../molecules/DropdownMenu/DropdownMenu.Trigger';
+import { DropdownMenuContent } from '../../molecules/DropdownMenu/DropdownMenu.Content';
+import { UserAvatar } from '../UserAvatar/UserAvatar';
+
+export type Assignee = {
```
Comment (Omri-Levy) [inline apps/backoffice-v2/src/common/components/atoms/AssignDropdown/AssignDropdown.tsx]: Let's use `Pick` on the `User` type here instead.

File: services/workflows-service/src/filter/dtos/temp-zod-schemas.ts [hunk 2]
```diff
@@ -196,7 +196,7 @@ export const WorkflowRuntimeDataWhereInputSchema = z.lazy(() =>
       assigneeId: zStringNullableFilterStringNullUnion.optional(),
       config: z.unknown().optional(),
       state: zStringNullableFilterStringNullUnion.optional(),
-      tags: z.any().optional(),
+      tags: z.any(),
```
Comment (tomer-shvadron) [inline services/workflows-service/src/filter/dtos/temp-zod-schemas.ts:199]: @Omri-Levy reminder

File: apps/backoffice-v2/src/common/components/atoms/icons/index.tsx [hunk 3]
```diff
@@ -455,6 +456,39 @@ export const PhotoSvg = (props: SVGProps<SVGSVGElement>) => {
   );
 };
 
+export const CheckFolderSvg = (props: SVGProps<SVGSVGElement>) => (
```
Comment (Omri-Levy) [inline apps/backoffice-v2/src/common/components/atoms/icons/index.tsx]: Let's make sure what is already available in `lucide-react`.

File: apps/backoffice-v2/src/common/utils/string-to-rgb/string-to-rgb.ts [hunk 4]
```diff
@@ -0,0 +1,42 @@
+const hashString = (s: string) => {
```
Comment (Omri-Levy) [inline apps/backoffice-v2/src/common/utils/string-to-rgb/string-to-rgb.ts]: Let's avoid sub 3 characters names. `str` is fine.

File: apps/backoffice-v2/src/pages/Entity/components/Case/interfaces.ts [hunk 5]
```diff
@@ -11,8 +11,7 @@ export interface IItemProps {
   fullName: string;
   createdAt: string;
   avatarUrl: string;
-  assigneeId: string;
-  assigneeFullName: string;
+  assignee: { id: string; fullName: string };
```
Comment (Omri-Levy) [inline apps/backoffice-v2/src/pages/Entity/components/Case/interfaces.ts]: Use `Pick` here on `User` as well.

File: apps/backoffice-v2/src/pages/Entity/components/Case/hooks/useCaseActionsLogic/useCaseActionsLogic.tsx [hunk 6]
```diff
@@ -78,13 +78,18 @@ export const useCaseActionsLogic = ({ workflowId, fullName }: IUseActions) => {
   }, [workflow]);
 
   const isActionButtonDisabled = !caseState.actionButtonsEnabled;
-  const onTriggerAssignToMe = true;
   const documentsToReviseCount = workflow?.context?.documents?.filter(
     document => document?.decision?.status === 'revision',
   )?.length;
 
+  const assignedUser = workflow?.assignee
```
Comment (Omri-Levy) [inline apps/backoffice-v2/src/pages/Entity/components/Case/hooks/useCaseActionsLogic/useCaseActionsLogic.tsx:85]: Let's revisit this tomorrow.

File: apps/backoffice-v2/src/pages/Entity/components/Case/Case.Actions.tsx [hunk 7]
```diff
@@ -38,63 +38,51 @@ import { tagToBadgeData } from './consts';
...
               id: authenticatedUser?.id,
               fullName: authenticatedUser?.fullName,
             },
+            ...((assignees ?? []) as Assignee[]),
           ]}
-          authenticatedUser={authenticatedUser}
-          caseState={caseState}
           onAssigneeSelect={id => {
-            onMutateAssignWorkflow(id, onTriggerAssignToMe);
+            onMutateAssignWorkflow(id, id === authenticatedUser?.id);
```
Comment (Omri-Levy) [inline apps/backoffice-v2/src/pages/Entity/components/Case/Case.Actions.tsx:78]: Let's revisit this flag.

File: apps/backoffice-v2/src/common/components/atoms/UserAvatar/UserAvatar.tsx [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+import { ctw } from '../../../utils/ctw/ctw';
+import React from 'react';
+
+interface IUserAvatarProps {
+  fullName: string;
+  avatarUrl?: string;
+  className?: string;
+}
+
+export const UserAvatar: React.FC<IUserAvatarProps> = ({ avatarUrl, fullName, className }) => (
```
Comment (Omri-Levy) [inline apps/backoffice-v2/src/common/components/atoms/UserAvatar/UserAvatar.tsx:12]: It seems we either don't need a component or it's a naming issue.

File: apps/backoffice-v2/src/pages/Entity/components/Case/Case.Actions.tsx (hunk 7 above)
Comment (tomer-shvadron) [reply apps/backoffice-v2/src/pages/Entity/components/Case/Case.Actions.tsx:78]: We should fix it in a separate (*cough cough* tech debt *cough cough*) issue

**Pull request merged**